lib/db/schema/patch-039-040.sql

```diff
@@ -0,0 +1,97 @@
+-- Drop the old reminders table, it was not functioning correctly
+DROP TABLE IF EXISTS `verificationReminders`;
```

Comment:
@jrgm can you please confirm that reminders are disabled in production, so dropping this table will not cause account creations to error out when attempting to add a reminder?  I'm pretty sure that's the case, but I want to be *very* sure.